README.md

```diff
@@ -89,7 +89,7 @@ with the operator of the LAB_RESULTS instance for the correct value.
 The following three variables control how often the adaptor performs a MESH polling cycle. During a polling cycle the 
 adaptor will download and acknowledge up to "the first 500 messages" (a MESH API limit).
 
-Important: If the MESH mailbox uses workflows other than `LAB_RESULTS_REG` and `LAB_RESULTS_RECEP` then these messages must be
+Important: If the MESH mailbox uses workflows other than `PATH_MEDRPT_V3`, `PATH_MEDRPT_V3_ACK`, `SCRN_BCS_MEDRPT_V4` and `SCRN_BCS_MEDRPT_V4_ACK` then these messages must be
```

Comment:
Can we add some hard wraps so the line isn't so long when reading the raw file? Seems to be about 120 chars max in most of this file too